router/courses.js

```diff
@@ -61,17 +57,20 @@ router.get('/courses/', function (request, response) {
             var form = '';
             var createClassBtn = '';
             var userInfo = request.session.passport.user;
-            // Super user hack
-            if (userInfo.mailId === 'mahipal2@illinois.edu') {
+            
+            // Add create-a-class section if user is authenticated
+            // Super user hack            
+            if (userInfo.mailId === 'mahipal2@illinois.edu' || userInfo.mailId === 'testuser@illinois.edu') {
                 form = getCreateClassForm(userInfo);
                 createClassBtn =
                     '<button class="btn" data-toggle="modal" data-target="#createPanel">' +
                     '          Create a New Class</button>';
             }            
+            permission.isManagingAllowed(userid, 'fc51b78d-f414-45b6-8ef5-95c9c61aa713');
```

Comment:
What's this doing?

Reply:
Whoops, might be a line that I forgot to delete after testing.